### DISCUSSION TITLE
Cant see files/folders/isos on main ventoy partition

### DISCUSSION THREAD

**Discussion by Tesserath** [ventoy/Ventoy#1123] [Q&A]
I really need help with this. The data still there and GB's are beeing ocupped but I CANT SEE ANYTHING. Even my custom boot theme is gone. Iam not worried about isos but i have important data there that is hidden. Obviusly i tried to see hidden folders but nothing there. 
![Captura de pantalla 2021-10-07 144842](https://user-images.githubusercontent.com/92098313/136389417-e0bcfb2e-6dfe-49e2-8607-3cc3b566867b.png)

**Comment by steve6375** [comment]
You probably have a fake USB drive.
These are drives which lie about their capacity
For example, you have a 32GB USB drive but it contains only 8GB of memory.
Once you fill it past 8GB by copying large ISOs on, it starts to overwrite the previous data.
https://rmprepusb.com/tutorials/007-all-about-fake-sd-cards-and-usb-flash-drives/
You could try a program like TestDIsk (there is a Windows version) and then PhotoRec - and try to copy off  any files you want to rescue.
https://rmprepusb.com/tutorials/028-recover-files-and-photos-from-a-corrupt-sd-card-or-any-drive-free/
However, you have probably lost everything...

**Reply by Tesserath** [reply]
Thanks for the reply but thats not problem. As i am using genuine kingston memory sticks. But i think i found the problem. Its probably caused by a virus that modify attrib files and hide them. That explains ventoy still working when i boot from it, but the main folder is empty.
